plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -153,7 +155,11 @@ static void installAppEngineRuntime(IFacetedProject project, IProgressMonitor mo
   }
 
   private static String findCloudSdk() {
-    return PathResolver.INSTANCE.getCloudSdkPath().toString();
+    File location = CloudSdkProvider.getDefaultSdkLocation();
```

Comment:
We don't want the default location. We want the actual location. If that's what the method provides, then we should rename it to just getLocation.